edx_proctoring/templates/timed_exam/footer.html

```diff
@@ -2,7 +2,7 @@
 <div class="footer-sequence">
   <h4> {% trans "Can I request additional time to complete my exam?" %} </h4>
     <p>{% blocktrans %}
-      If you have disabilities or are taking the exam in difficult conditions,
+      If you have disabilities,
```

Comment:
Yeah, there should be, but I'm not 100% what those steps are in this repo. It will need to be addressed when someone is ready to spend more than "5 minutes during grooming" amount of effort on this work.